src/Command/Auth/VerifyPhoneNumberCommand.php

```diff
@@ -105,16 +105,17 @@ protected function execute(InputInterface $input, OutputInterface $output)
         });
 
         $this->debug('Refreshing phone verification status');
-        $needsVerify = $httpClient->post( '/me/phone?force_refresh=1')->json()['verify_phone'];
+        $needsVerify = $httpClient->post( '/me/verification?force_refresh=1')->json();
         $this->stdErr->writeln('');
 
-        if ($needsVerify) {
-            $this->stdErr->writeln('Phone verification succeeded but the status check failed.');
-            return 1;
+        if ($needsVerify['state']) {
+            if ($needsVerify['type'] == 'phone') {
+                $this->stdErr->writeln('Phone verification succeeded but the status check failed.');
+                return 1;
+            }
```

Comment:
Maybe this whole check is unnecessary, as this should never happen, right? and perhaps it will happen because computers... but the customer can't do much about it?

At least if we keep it I'd tweak it as follows, as I think the 'state' check wouldn't matter:

```suggestion
        if ($needsVerify['type'] == 'phone') {
            $this->stdErr->writeln('Phone verification succeeded but the status check failed.');
            return 1;
        }
```

Removing it would be deleting lines 106-116.